instances/technique/electrocardiography.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/electrocardiography",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": "Electrocardiography is the process of producing an electrogram of the heart: a graph of voltage versus time of the electrical activity of the heart using electrodes placed on the skin.",
```

Comment:
suggestion to change to:

"An 'electrocardiography' is a non-invasive technique used to record the electrical activity of a heart using electrodes placed on the skin. [adapted from [wikipedia](https://en.wikipedia.org/wiki/Electrocardiography)]"

Note: Please always add the reference from where you took / adapted the definition from if it is not your own phrasing.

Reply:
I updated the definition. The word electrocardiography is typically used without an indefinite article in English when we talk about the technique in general and not a concrete case, unless coupled with another word, e.g. electrocardiography measurement, electrocardiography system, electrocardiography score etc. So I skipped the "an" at the beginning.